src/igvfd/types/sample.py

```diff
@@ -107,6 +131,32 @@ class InVitroSystem(Biosample):
     item_type = 'in_vitro_system'
     schema = load_schema('igvfd:schemas/in_vitro_system.json')
 
+    @calculated_property(
+        schema={
+            'title': 'Summary',
+            'type': 'string',
+            'notSubmittable': True,
+        }
+    )
+    def summary(self, request, biosample_term, taxa, classification, time_post_factors_introduction=None, time_post_factors_introduction_units=None):
+        sample_term_object = request.embed(biosample_term, '@@object?skip_calculated=true')
+        term_name = sample_term_object.get('term_name')
+        term_and_classification = f'{term_name} {classification}'  # default case
+        if classification in term_name:
+            term_and_classification = term_name  # if classification in term_name, drop classification from summary
+        elif 'cell' in classification and 'cell' in term_name:
+            term_name = term_name.replace('cell', classification)
```

Comment:
Minor suggestion, but this can be combined with the following line instead of splitting it in two. (so "term_and_classification = term_name.replace(...". Likewise on line 150.